compiler/test/stdlib/list.test.gr

```diff
@@ -41,6 +41,11 @@ let multiply = (n, i) => n * i
 assert mapi(multiply, numbers) == [0, 3, 8]
 assert mapi(multiply, []) == []
 
+# List.flatMap
+
+assert flatMap(x => [x + 1], list) == [2, 3, 4]
```

Comment:
Would maybe be good to also test having multiple items in the list:
```suggestion
assert flatMap(x => [x + 1], list) == [2, 3, 4]
assert flatMap(x => [x, x + 1], list) == [1, 2, 2, 3, 3, 4]
```

Reply:
No idea how I missed this case 🤦